.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      env: MATRIX_EVAL="CC=gcc-5 && CXX=g++-5"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-5
+    - os: linux
+      env: MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
+      addons:
+        apt:
+          sources:
+            - llvm-toolchain-trusty-4.0
+            - ubuntu-toolchain-r-test
+          packages:
+            - clang-4.0
+            - libstdc++-5-dev
+    - os: linux
+      env: MATRIX_EVAL="CC=clang-5.0 && CXX=clang++-5.0"
+      addons:
+        apt:
+          sources:
+            - llvm-toolchain-trusty-5.0
+            - ubuntu-toolchain-r-test
+          packages:
+            - clang-5.0
+            - libstdc++-5-dev
+
+# prevent Travis from overwriting our CXX variables
+before_install:
+    - eval "${MATRIX_EVAL}"
+    - echo $CXX
+
+script:
+    - mkdir build
+    - cd build
+    - cmake --config Debug ..
```

Comment:
appveyor uses release config but travis debug?

Reply:
Yes, in case of errors in tests, I think the Linux build should give you nice stack trace.

Reply:
But that's something you would want to verify.

Reply:
https://ci.appveyor.com/project/janisozaur/acl/build/job/0jr9jr70rxa6au4t and https://travis-ci.org/janisozaur/acl/jobs/306959728